modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -73,6 +73,17 @@ REMOVE n[$prop]
 
 | Added the ability to dynamically reference properties in xref:clauses/set.adoc#set-dynamically-a-property[SET] and xref:clauses/remove.adoc#remove-remove-a-property-dynamically[REMOVE] clauses.
 
+a|
+label:functionality[]
+label:new[]
```

Comment:
should this be in the new or the updated category? It is new functionality on an existing command

Reply:
Looking at some recent examples. adding gs:// to LOAD CSV and adding CIDR ranges to GRANT/DENY were both New, so I think it should be new.